sdk/core/amqp-common/README.md

```diff
@@ -47,12 +41,12 @@ This SDK houses core AMQP common related functionality in use bu Azure SDKs that
 
 ## Examples
 
-Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/amqp/samples) directory for detailed samples.
+Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/amqp-common/samples) directory for detailed samples.
```

Comment:
@ramya0820 
Can you update that link to point to the release tag instead so that it will be permanent? https://github.com/Azure/azure-sdk-for-js/tree/%40azure/amqp-common_1.0.0-preview.4/sdk/core/amqp-common/samples

Reply:
similarly we should set the package homepage to that link.